### PR TITLE
Update Mangaworld.kt

### PR DESCRIPTION
**Pull request by vpervenditti**
fix mangaworld url

### CODE REVIEW

**Comment by arkon** [comment]
Update the version number in build.gradle as well.

**Comment by vpervenditti** [comment]
> Update the version number in build.gradle as well.

done

File: src/it/mangaworld/build.gradle [hunk 1]
```diff
@@ -3,10 +3,10 @@ apply plugin: 'kotlin-android'
 
 ext {
     appName = 'Tachiyomi: Mangaworld'
-    pkgNameSuffix = 'it.mangaworld'
+    pkgNameSuffix = 'tv.mangaworld'
```
Comment (arkon) [inline src/it/mangaworld/build.gradle:6]: Don't change this.

File: src/it/mangaworld/build.gradle [hunk 2]
```diff
@@ -3,10 +3,10 @@ apply plugin: 'kotlin-android'
...
     extClass = '.Mangaworld'
     extVersionCode = 1
```
Comment (arkon) [inline src/it/mangaworld/build.gradle:8]: Should be this one.

File: src/it/mangaworld/build.gradle [hunk 3]
```diff
@@ -3,10 +3,10 @@ apply plugin: 'kotlin-android'
...
-    libVersion = '1.2'
+    libVersion = '1.3'
```
Comment (arkon) [inline src/it/mangaworld/build.gradle:9]: Don't change this.

**Comment by arkon** [comment]
Did the website layout change? If so it'll require more changes/testing.

Closes #2618.

**Comment by vpervenditti** [comment]
> Did the website layout change? If so it'll require more changes/testing.
> 
> Closes #2618.

as i know, no. They just changed the domain.

**Comment by vpervenditti** [comment]
> Did the website layout change? If so it'll require more changes/testing.
> 
> Closes #2618.

is everything okay, now?

**Comment by arkon** [comment]
You need to update `extVersionCode`

**Comment by vpervenditti** [comment]
> `extVersionCode`

done.

**Pull request merged**